integration/token/fungible/support.go

```diff
@@ -884,6 +885,20 @@ func GetTMS(network *integration.Infrastructure, networkName string) *topology.T
 	return tms
 }
 
+func GetTMSByAlias(network *integration.Infrastructure, alias string) *topology.TMS {
```

Comment:
Maybe it would be better to use a type

```
type Alias string
```
to avoid confusion with the network

Reply:
okay